Dockerfile

```diff
@@ -17,8 +17,10 @@ RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-w -s' -a -installsuffix cgo
 
 
 FROM alpine:3.7
-ENV XDG_CONFIG_HOME=/che-jwtproxy-config/
-VOLUME /che-jwtproxy-config
 COPY --from=builder /go/src/github.com/eclipse/che-jwtproxy/jwtproxy /usr/local/bin
 ENTRYPOINT ["jwtproxy"]
-CMD ["-config", "/che-jwtproxy-config/config.yaml"]
+# The JWT proxy needs 2 things:
+# * the location of the configuration file supplied as an argument:
+#   `-config <location/of/the/config.yaml>`
+# * The XDG_CONFIG_HOME environment variable pointing to a directory where to store auth keys
```

Comment:
I see `XDG_CONFIG_HOME` var name the first time, and it probably because I'm not a big expert in Linux systems...
Actually JWT proxy does not need it, but it's one of options on how to configure base directory.
As a bit simpler alternative we could describe here that HOME should be specified 
https://github.com/eclipse/che-jwtproxy/blob/3f2cb18666b6d0f3185ead7a85e026379528c2c7/jwt/privatekey/autogenerated/autogenerated.go#L151

Reply:
not sure if setting `HOME` would not have some other consequences... I was trying to keep the same semantics as was in the original Dockerfile.